src/core/etl/src/Flow/ETL/DSL/Entry.php

```diff
@@ -221,6 +223,26 @@ final public static function list_of_string(string $name, array $value) : RowEnt
         return new RowEntry\ListEntry($name, ListElement::string(), $value);
     }
 
+    /**
+     * @param array<string> $value
+     *
+     * @throws InvalidArgumentException
+     */
+    final public static function map_of_ints(string $name, array $value) : RowEntry
+    {
+        return new RowEntry\MapEntry($name, MapKey::integer(), MapValue::integer(), $value);
+    }
+
+    /**
+     * @param array<string> $value
+     *
+     * @throws InvalidArgumentException
+     */
+    final public static function map_of_string(string $name, array $value) : RowEntry
```

Comment:
technically speaking this is not map of string, but map_of_string_to_string - lets think more about those factory methods once we get all pieces in place